### PR TITLE
lighthouse 2.1.0

### PR DESCRIPTION
**Pull request by realbigsean**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/lighthouse.rb [hunk 1]
```diff
@@ -15,6 +15,7 @@ class Lighthouse < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "llvm" => :build
```
Comment (bayandin) [inline Formula/lighthouse.rb:18]: Why `llvm` is required?

Comment (realbigsean) [reply Formula/lighthouse.rb:18]: The new lighthouse version requires `libclang` which I thought was inlcuded in `llvm`.  But adding this dependency might only be necessary on Ubuntu. This prior run without `llvm` passed on ARM but failed on Ubuntu: https://github.com/Homebrew/homebrew-core/runs/4887822813?check_suite_focus=true

```
    thread 'main' panicked at 'Unable to find libclang: "couldn't find any valid shared libraries matching: ['libclang.so', 'libclang-*.so', 'libclang.so.*', 'libclang-*.so.*'], set the `LIBCLANG_PATH` environment variable to a path where one of these files can be found (invalid: [])"', /github/home/.cache/Homebrew/cargo_cache/registry/src/github.com-1ecc6299db9ec823/bindgen-0.59.2/src/lib.rs:2144:31
```

Comment (bayandin) [reply Formula/lighthouse.rb:18]: Ah, right, in this case, it should be wrapped in `on_linux` block:

```
  on_linux do
    depends_on "llvm"
  end
```

Comment (realbigsean) [reply Formula/lighthouse.rb]: Thanks! Trying that out now

**Review by iMichka — APPROVED** [review]

**Comment by iMichka** [comment]
Thanks @realbigsean !

**Pull request closed without merging**